src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.ts

```diff
@@ -264,3 +353,28 @@ export function fromStreamName<T extends StreamType>(
     streamId: parts[1],
   };
 }
+
+/**
+ * Converts the `stream.metadata.streamPosition` of the given `stream`
+ * to a `string` value to be stored in MongoDB
+ */
+function streamPositionSerializer<Stream extends EventStream>(
+  stream: Stream,
+): Stream {
+  // @ts-expect-error serializing as a `string`
+  stream.metadata.streamPosition = stream.metadata.streamPosition.toString();
```

Comment:
**CONCERN:** Wouldn't be enough to use [useBigInt64 option](https://mongodb.github.io/node-mongodb-native/5.2/interfaces/MongoClientOptions.html#useBigInt64)? Read also more [here](https://www.mongodb.com/community/forums/t/mongodb-nodejs-driver-5-1-0-released/214717).

Reply:
I didn't know about that option, I may have been seeing some outdated forums saying string was best

Reply:
To be fair, I also found that option today (tho, I knew that BSON on its own can handle BigInts).  😅